enable/gcbench/suite.py

```diff
@@ -137,3 +149,63 @@ def __call__(self):
                 self.gc.set_text_position(4, y)
                 self.gc.show_text(line)
                 y -= self.font.size * 1.4
+
+
+class draw_path_linear_gradient:
+    def __init__(self, gc, module):
+        self.gc = gc
+
+    def __call__(self):
+        import numpy as np
+        # colors are 5 doubles: offset, red, green, blue, alpha
+        starting_color = np.array([0.0, 1.0, 1.0, 1.0, 1.0])
+        ending_color = np.array([1.0, 0.0, 0.0, 0.0, 1.0])
```

Comment:
I would move this setup of the gradient array out of `__call__` and into `__init__`.